PxtlCa.XmlCommentMarkDownGenerator/TagRenderers.cs

```diff
@@ -29,6 +29,14 @@ public TagRenderer(string formatString, Func<XElement, ConversionContext, IEnume
                         x.Element("members").Elements("member").ToMarkDown(context.MutateAssemblyName(x.Element("assembly").Element("name").Value))
                 }
             ),
+            ["br"] = new TagRenderer(
+                "\n",
+                (x, context) => new string[0]
+            ),
+            ["seealso"] = new TagRenderer(
+                "##### See also: {0}\n",
```

Comment:
If possible (and consider this just a suggestion):

https://gist.github.com/asabaylus/3071099

Github-flavoured markdown automatically generates an anchor for each heading.  That means that seealso could be an intra-document link.